pyproject.toml

```diff
@@ -23,14 +27,15 @@ classifiers = [
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
     "Typing :: Typed",
 ]
 dynamic = ["version"]
-dependencies = []
+dependencies = [
+    "tenacity>=8.5.0",
+    "loguru>=0.7.2",
+]
```

Comment:
I think we are missing `httpx` as a dependency here?

Reply:
Good catch, thank you! I guess I did not see any problems since I was installing the dev dependencies, which include pytest-httpx, which pulls in httpx.